.cicd/platforms/unpinned/ubuntu-20.04-unpinned.dockerfile

```diff
@@ -3,7 +3,7 @@ ENV VERSION 1
 # install dependencies.
 RUN apt-get update && \
     apt-get upgrade -y && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y git make \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y git make libtinfo5 \
     bzip2 automake libbz2-dev libssl-dev doxygen graphviz libgmp3-dev \
     autotools-dev python2.7 python2.7-dev python3 python3-dev python3-pip python3-requests \
     autoconf libtool curl zlib1g-dev sudo ruby libusb-1.0-0-dev libpq-dev postgresql postgresql-server-dev-all \
```

Comment:
**Optional** - Since we're touching these package lists, it would be great if we could clean them up by putting one package on each line in alphabetical order instead of following the existing anti-pattern where it is just a jumbled mess. This transform will do it for you:
```bash
pbpaste | rev | sed -E 's_^_\\_' | rev | tr -d '\\\n' | sed 's/ \+/ /g' | sed 's/^/ /' | sed 's/ / \\\n        /g' | sort -d | uniq | head -c -3 | pbcopy
```
Just copy the multi-line list of packages (excluding `apt-get install -y` and the `&& \` at the very end), run that transform, then paste.

Reply:
Done.